### DISCUSSION TITLE
add enableBasicSetMapping compat flag to wadwaz-1 and wapirz-1

### DISCUSSION THREAD

**Discussion by slikone27** [zwave-js/zwave-js#7047] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
wadwaz-1 and wapirz-1 are getting added with basic sensors of on/off (light bulb icon) instead of open/close or motion.  the only thing that changes on open or motion is alarmLevel from "0" to "255"

What did you expect to happen instead?
them to get added correct

add the devices via zwave JS

Anything else we should know?
this apparently was fixed in an older version:
https://github.com/zwave-js/node-zwave-js/pull/2256

### Software versions

Driver version:
12.11.2
Server version:
1.36.0
Home ID:
3437780000
Server URL:
ws://core-zwave-js:3000

using home assistant

### Device information

Manufacturer: GoControl / Linear
Model name: see below
Node ID: WAPIRZ-1 = 19 / WADWAZ-1 = 20

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[home-assistant_zwave_js_2024-07-26T00-24-24.511Z.log](https://github.com/user-attachments/files/16385153/home-assistant_zwave_js_2024-07-26T00-24-24.511Z.log)

**Comment by AlCalzone** [comment, marked as answer]
Your log shows that the binary sensor is getting updated:
```
2024-07-26T00:21:29.173Z CNTRLR   [Node 020] [~] [Binary Sensor] Any: true => false                 [Endpoint 0]
```
AFAIK HA hides those by default if Notification CC is supported, because that CC is better suited. But in this case, Notification CC is implemented strangely, so it does not help. Enable the Binary sensor entity and it should work.

**Reply by slikone27** [reply]
Thanks I meant to reply earlier... Someone else helped me on the HA forums as well.  But they said to leave this open because it should be getting hidden by default and it is not.  But I do have the sensors working now and have manually hidden the basic CC ones.

"Z-Wave JS should hide the Basic CC, still a bug."

**Reply by slikone27** [reply]
should an issue be created for: "Z-Wave JS should hide the Basic CC"?

**Reply by AlCalzone** [reply]
Have you re-interviewed the device? There were a couple of fixed related to hiding unnecessary Basic CC entities recently.

**Reply by slikone27** [reply]
yes re-interviewed it multiple times no change.  doesn't appear to be hiding it by default anymore.  also added about 15 more of those devices and none of them were hidden.

**Reply by AlCalzone** [reply]
Do you have Z-Wave JS UI too? There you could export a node dump  that helps figure out why.

**Reply by slikone27** [reply]
no just the standard HA integration

**Reply by AlCalzone** [reply]
Ok, can you make another log please, this time including a re-interview?
